drivers/rf-transceiver/hmc630x/hmc630x.c

```diff
@@ -246,6 +285,14 @@ int hmc630x_remove(struct hmc630x_dev *dev)
 	return 0;
 }
 
+enum hmc630x_type hmc630x_type(struct hmc630x_dev *dev)
+{
+	if (dev)
+		return dev->type;
+	else
+		return -EINVAL;
+}
+
```

Comment:
Well, yes, but accessing it would also give users write access and i don't want this for this driver.
As a matter of fact, i think this should be the case for all drivers... 
Or at least have a public and a private structure, but i'd rather not go that way for 1 single member.